### PR TITLE
dash/changelog-improvements

### PR DESCRIPTION
**Pull request by sebastianbochan**
Adapted changelog to dashboards requiremenets.

### CODE REVIEW

File: changelog/pr-log.js [hunk 1]
```diff
@@ -50,20 +50,37 @@ const parseUpgradeNotes = p => {
 
 const loadPulls = async (
     since,
-    branches = 'master'
+    branches = 'master',
+    isDashboards
```
Comment (bre1470) [inline changelog/pr-log.js:54]: It might be useful to have a tag parameter here.

File: changelog/pr-log.js [hunk 2]
```diff
@@ -50,20 +50,37 @@ const parseUpgradeNotes = p => {
...
 ) => {
     let page = 1;
     let pulls = [];
+    let lastTagSha;
 
     const tags = await octokit.repos.listTags({
```
Comment (bre1470) [inline changelog/pr-log.js:60]: Then this can integrate the tag argument from the new code below and we avoid dashboard-specific code.

**Comment by sebastianbochan** [comment]
I would say, that this is a PoC on how we can proceed, but I'm open to suggestions.

**Review by bre1470 — APPROVED** [review]

**Review by karolkolodziej — APPROVED** [review]

**Review by goransle — APPROVED** [review]

**Review by Vazonik — APPROVED** [review]

**Review by TorsteinHonsi — APPROVED** [review]
Looks good to me!

**Pull request merged**